crates/starknet/src/starknet/get_class_impls.rs

```diff
@@ -129,9 +135,23 @@ mod tests {
         assert_eq!(class_hash, expected);
     }
 
+    #[test]
+    fn get_class_hash_at_generated_accounts_without_state_archive() {
+        let (mut starknet, account) = setup(Some(100000000), StateArchiveCapacity::None);
+
+        starknet.generate_new_block(StateDiff::default(), None).unwrap();
+        starknet.generate_pending_block().unwrap();
+
+        let block_number = starknet.get_latest_block().unwrap().block_number();
+        let block_id = BlockId::Number(block_number.0);
+
+        let class_hash = starknet.get_class_hash_at(block_id, account.account_address);
+        assert!(class_hash.is_err());
```

Comment:
Can you assert the error type? So that we're sure it's not failing with something unforeseen

Reply:
Fixed in c727b6cd25e1aaa1debbc6ded48ac22872545c48